api/envoy/config/trace/v2/trace.proto

```diff
@@ -80,6 +80,14 @@ message DynamicOtConfig {
   google.protobuf.Struct config = 2;
 }
 
+// Configuration for the Datadog tracer.
+message DatadogConfig {
+  // The cluster to use for submitting traces to the Datadog agent.
+  string collector_cluster = 1 [(validate.rules).string.min_bytes = 1];
+  string service_name = 2 [(validate.rules).string.min_bytes = 1];
+  bool priority_sampling = 3;
```

Comment:
@rnburn Support for that tag is being added in https://github.com/DataDog/dd-opentracing-cpp/pull/59 should be ready today. 

@cgilmour We should copy/have similar comments as https://github.com/DataDog/dd-opentracing-cpp/blob/master/src/tracer_factory.cpp#L21 We should also default priority sampling to on here (whereas normally it's off by default), since envoy uses it.